patchwork-level-generators/README.md

```diff
@@ -0,0 +1,2 @@
+## TODO
```

Comment:
Might want to add some detail as to what this module does

Reply:
I added more to the file, not sure why it didn't update